api/v1/swagger/paths/holdings.json

```diff
@@ -0,0 +1,352 @@
+{
+  "/holdings": {
+    "get": {
+      "x-mojo-to": "Holding#list",
+      "operationId": "listHoldings",
+      "tags": ["patrons", "holdings"],
+      "parameters": [
+        {
+          "name": "holding_id",
+          "in": "query",
+          "description": "Case insensitive starts-with search for internal holding identifier",
+          "type": "integer"
+        },
+        {
+          "name": "biblionumber",
+          "in": "query",
+          "description": "Case insensitive starts-with search for internal biblio identifier",
+          "type": "integer"
+        },
+        {
+          "name": "biblioitemnumber",
+          "in": "query",
+          "description": "Case insensitive starts-with search for internal biblio item identifier",
+          "type": "integer"
+        },
+        {
+          "name": "frameworkcode",
+          "in": "query",
+          "description": "Case insensitive starts-with search for framework used in cataloging this record",
+          "type": "string"
+        },
+        {
+          "name": "holdingbranch",
+          "in": "query",
+          "description": "Case insensitive starts-with search for internal biblio identifier for library that is currently in possession of this holding",
+          "type": "string"
+        },
+        {
+          "name": "location",
+          "in": "query",
+          "description": "Case insensitive starts-with search for authorized value for the shelving location for this holding",
+          "type": "string"
+        },
+        {
+          "name": "ccode",
+          "in": "query",
+          "description": "Case insensitive starts-with search for authorized value for the collection code for this holding",
+          "type": "string"
+        },
+        {
+          "name": "callnumber",
+          "in": "query",
+          "description": "Case insensitive starts-with search for call number for this holding",
+          "type": "string"
+        },
+        {
+          "name": "suppress",
+          "in": "query",
+          "description": "Case insensitive starts-with search for boolean indicating whether the record is suppressed in OPAC",
+          "type": "integer"
+        },
+        {
+          "name": "timestamp",
+          "in": "query",
+          "description": "Case insensitive starts-with search for date and time this recird was last altered",
```

Comment:
This is a pretty nice idea.
Thanks to ISO 8601 we can easily filter by year-month-day